.php-cs-fixer.php

```diff
@@ -5,6 +5,8 @@
 $config = new PhpCsFixer\Config();
 $config->setCacheFile(__DIR__ . '/php-cs-fixer.cache');
 $config->setFinder($finder);
+$config->setIndent("\t");
+$config->setLineEnding("\n");
 $config->setRules([
         '@PSR2' => true,
```

Comment:
I believe PSR12 deprecates PSR2 as of a few years now, maybe we can update this? (PHP-CS-Fixer might just not  have updated the name, not sure)

Reply:
Changed and updated to PSR12 standard.

Reply:
Do we also need to update the PSR-4 to PSR-12 for the composer autoload?

Reply:
Thanks! PSR-4 is still current, PSR-12 deals only with extended code styles (each PSR deals with different subjects, rather than the latest/highest number being the most current, similar to Python's PEPs, if you're familiar)

Reply:
Ok, then I'll leave it in the composer unchanged. :)